test/uk/gov/hmrc/helptosavestridefrontend/controllers/StrideControllerSpec.scala

```diff
@@ -31,7 +32,7 @@ import uk.gov.hmrc.helptosavestridefrontend.models.CreateAccountResult.AccountCr
 import uk.gov.hmrc.helptosavestridefrontend.models.EnrolmentStatus.{Enrolled, NotEnrolled}
 import uk.gov.hmrc.helptosavestridefrontend.models.eligibility.{EligibilityCheckResponse, EligibilityCheckResult}
 import uk.gov.hmrc.helptosavestridefrontend.models.{CreateAccountResult, EnrolmentStatus, NSIUserInfo}
-import uk.gov.hmrc.helptosavestridefrontend.util.{NINO, Result ⇒ _}
+import uk.gov.hmrc.helptosavestridefrontend.util.{Result ⇒ _, _}
```

Comment:
you can replace with line with: `import uk.gov.hmrc.helptosavestridefrontend.util.NINO`